inference-engine/src/gna_plugin/optimizer/gna_pass_manager.cpp

```diff
@@ -268,6 +269,40 @@ void HandleMultipleActivationsForTheLayerPass::run() {
     }
 }
 
+void ForbidActivationFusingPass::run() {
```

Comment:
This pass could be implemented as part of InsertDiagonalLayerPass

Reply:
It was done like that in the first approach. Then I have added new pass to make the code separated for better maintainability and not complicate InsertDiagonalLayerPass() more.